questionnaire/form-helper.js

```diff
@@ -8,6 +8,14 @@ const templateEngineService = createTemplateEngineService();
 const {render} = templateEngineService;
 const shouldShowSignInLink = require('./utils/shouldShowSignInLink');
 
+function checkIsContextPage(sectionId) {
+    return (
+        sectionId.includes('-context-') ||
+        sectionId === 'p-applicant-you-cannot-get-compensation' ||
+        sectionId === 'p--before-you-continue'
+    );
+}
+
```

Comment:
Not a fan of hardcoding these ids in to CW. I have introduced a "pageType" in the tasklist work that we're doing. Could we do something like this instead?
https://github.com/ministryofjustice/cica-apply-web/commit/0f46737e03a430d387d755514d684bee4e9e5379

https://github.com/ministryofjustice/cica-apply-data-capture-service/commit/dcde46c7fa6fc25f10169723bf6e5df01ffe1e5d

Reply:
I have changed it to use page type as you did for the tasklist:
- https://github.com/ministryofjustice/cica-apply-data-capture-service/pull/467
- https://github.com/CriminalInjuriesCompensationAuthority/q-templates-application/pull/320